src/qminer/qminer_core.h

```diff
@@ -3219,46 +3219,45 @@ namespace TStreamAggrOut {
 
 	class IInt {
 	public:
-		// retireving value from the aggregate
+		// retrieving value from the aggregate
 		virtual int GetInt() const = 0;
 	};
 
 	class IFlt {
 	public:
-		// retireving value from the aggregate
+		// retrieving value from the aggregate
 		virtual double GetFlt() const = 0;
 	};
 
+	class ISparseVec {
```

Comment:
The problem arises from the fact that they both come from the same interface - `IValVec<>` - just with different template parameters. Since there are no parameters, they differ only in return type. And compiler can't handle that.